### PR TITLE
Issue #50

### PR DESCRIPTION
**Pull request by kvhooreb**

### CODE REVIEW

**Comment by kvhooreb** [comment]
1. The onnx file should be a pkg_resource, like I mentionned on sleck.
2. `raymon/tests/profiling/models/retinopathy@2.0.0.json` and   `aymon/tests/profiling/retinopathy_data/3/trainLabels.csv` have no use. Delete them.
3. The unit test is not good, it should do more tests, like test the building of the extractor, test the amount of clusters, text whether the extractor can be serialized correctly and loaded correctly etc.

File: raymon/profiling/extractors/vision/anomaly.py [hunk 1]
```diff
@@ -1,70 +1,83 @@
...
-
-    def build(self, data, batch_size=16):
-        dataset = ImageDataset(loaded_data=data, transform=self.tfs)
-        # data is a list of images here
-        dataloader = DataLoader(dataset, shuffle=False, batch_size=batch_size, drop_last=True)
-        embeddings = torch.zeros(size=(len(data), 1000))
-        for batchidx, batchtf in enumerate(dataloader):
-            # batchtf = self.tfs(batch)
-            components = self.mobilenet(batchtf).detach()  # .numpy()
+            raise ValueError(f"type of data must be PIL.Image.Image, not {data.shape}")
```
[the remark this answers is not in the document]
Comment (mremreozan) [reply raymon/profiling/extractors/vision/anomaly.py:61]: It's done.

File: raymon/tests/profiling/test_DN2AnomalyScorer.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+from PIL import ImageFilter
+from PIL import ImageEnhance
+
+def test_DN2AnomalyScorer():
+    ImageFile.LOAD_TRUNCATED_IMAGES = True
+    ROOT = Path("..").resolve()
+    # test data contains 11 different images
+    TEST_DATA_PATH = ROOT / "profiling/retinopathy_data/3"
+    LIM = 500
```
[the remark this answers is not in the document]
Comment (mremreozan) [reply raymon/tests/profiling/test_DN2AnomalyScorer.py:23]: I think 10 images is enough to test the model. But we need more images to build the model, don't we?

File: raymon/tests/profiling/test_DN2AnomalyScorer.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            if n == lim:
+                break
+            img = Image.open(fpath)
+            img.thumbnail(size=(500, 500))
+            images.append(img)
+            metadata.append([{"name": "srcfile", "value": fpath.stem, "type": "label"}])
+        return images, metadata
+
+    # Load model, you can see how the model build.
+    MODEL_PATH = ROOT / "profiling/models/retinopathy@2.0.0.json"
```
[the remark this answers is not in the document]
Comment (mremreozan) [reply raymon/tests/profiling/test_DN2AnomalyScorer.py:39]: Building a model takes about 2 minutes with 318 images. Is it long? Should we decrease the number of images to build the model.

File: raymon/tests/profiling/test_DN2AnomalyScorer.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (kvhooreb) [reply raymon/tests/profiling/test_DN2AnomalyScorer.py]: Not for init testing the building, no

File: raymon/tests/profiling/test_DN2AnomalyScorer.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (kvhooreb) [reply raymon/tests/profiling/test_DN2AnomalyScorer.py]: 2 minutes is way too long. By decreasing the number of images, this building time will decrease.

**Pull request merged**